paddle/framework/CMakeLists.txt

```diff
@@ -35,6 +35,11 @@ py_proto_compile(framework_py_proto SRCS attribute.proto op_proto.proto op_desc.
 # Generate an empty __init__.py to make framework_py_proto as a valid python module.
 add_custom_target(framework_py_proto_init ALL COMMAND ${CMAKE_COMMAND} -E touch __init__.py)
 add_dependencies(framework_py_proto framework_py_proto_init)
+add_custom_command(TARGET framework_py_proto POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E make_directory ${PROJ_ROOT}/python/paddle/v2/framework/proto
+    COMMAND cp *.py ${PROJ_ROOT}/python/paddle/v2/framework/proto/
```

Comment:
[3357](https://github.com/PaddlePaddle/Paddle/pull/3357) have been tested with swig python. And I think copy all the generated file is not a good idea. If we have some other generated files, we have to modify this CMakeLists according to the change.